traffic_ops/traffic_ops_golang/deliveryservice/deliveryservices.go

```diff
@@ -606,7 +609,8 @@ func (ds *TODeliveryService) Read(h http.Header, useIMS bool) ([]interface{}, er
 		switch {
 		// NOTE: it's required to handle minor version cases in a descending >= manner
 		case version.Major > 3:
-			returnable = append(returnable, ds.RemoveLD1AndLD2())
+			dsV4 := ds.RemoveLD1AndLD2()
+			returnable = append(returnable, dsV4)
```

Comment:
this doesn't appear to change anything - was this supposed to be included in this PR?